pkg/storage/unified/sql/backend.go

```diff
@@ -763,6 +793,16 @@ func (b *backend) getHistory(ctx context.Context, req *resource.ListRequest, cb
 		listReq.MinRV = req.ResourceVersion
 	}
 
+	if listReq.MinRV == 0 && !listReq.Trash && req.VersionMatchV2 != resource.ResourceVersionMatchV2_Exact {
```

Comment:
Can you elaborate on what you're trying to cover with `req.VersionMatchV2 != resource.ResourceVersionMatchV2_Exact`?

Note that VersionMatchV2 might also be UNKNOWN. Generally with enums it's good to rather be explicit with the cases you want to cover.

Reply:
Okay. I'd recommend clarifying the intent with a comment then :)